package.json

```diff
@@ -185,7 +185,7 @@
     "chrome",
     "devtools"
   ],
-  "author": "The Chromium Authors",
+  "author": "Google Inc.",
```

Comment:
i think we do  "The Lighthouse Authors"

https://github.com/GoogleChrome/lighthouse/pull/10469